teemaderegister-be/src/models/user.js

```diff
@@ -11,7 +11,7 @@ const userSchema = new mongoose.Schema(
     profile: {
       firstName: { type: String, required: true },
       lastName: { type: String, required: true },
-      description: { type: String },
+      description: { type: String, default: '-' },
```

Comment:
In database still use undefined, do not save "-", under frontend if not desription is provided render "-" .Will add another comment with the solution